rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureWithCallback/rename_class_with_callback.php.inc

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback;
+
+use Exception;
+
+class BadRequest extends Exception {
+
+}
+
+$badRequest = new \Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback\BadRequest();
+
+if ($badRequest instanceof \Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback\BadRequest) {
+    echo \Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback\BadRequest::class;
+}
+
+class CorrectlyNamedException extends Exception {
+
+}
+
+$correctlyNamedException = new \Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback\CorrectlyNamedException();
+
+interface Processor {
+
+}
+
+class FruitProcessor implements \Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureWithCallback\Processor {
+
+}
+
```

Comment:
Could you split these to 1fixture file per case? That way its easier to read and possibly debug later :+1:

Reply:
Done :+1:

Do you have any suggestions regarding the configuration method? Any better idea than the `__callbacks__` key, or do you think its fine? I am also concerned about documentation, how should such configuration example be documented?

Reply:
You may need to move "__callbacks__" key to constant, eg: `RenameClassRector::CALLBACKS` so it more discoverable with IDE

Reply:
Done